### DISCUSSION TITLE
救命！我的朋友被困在空洞里了！

### DISCUSSION THREAD

**Discussion by 114pt76** [share121/inter-knot#4903] [委托]
我的那位朋友长这样
![Screenshot_20240915_164423_tv danmaku bili](https://github.com/user-attachments/assets/0b20f32e-38e3-4fa9-a7f2-ccb79e891918)

**Comment by 070612wwz** [comment]
我朋友（以骸）被一个待面具的男人追击了

**Comment by 114pt76** [comment]
你为什么有以骸朋友（恼）



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年09月15日 17:41 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 救命！我的朋友被困在空洞里了！ (Discussion #4903) |

我朋友（以骸）被一个待面具的男人追击了

**Reply by 070612wwz** [reply]
因为……因为我善＼(`Δ’)／
